test/verify/check-storage-nfs

```diff
@@ -93,6 +95,7 @@ class TestStorageNfs(StorageCase):
         self.dialog_set_val("dir", "/mounts/barbar")
         self.dialog_apply()
         self.dialog_wait_close()
+        self.addCleanup(m.execute, "umount /mounts/barbar && rmdir /mounts/barbar")
```

Comment:
Using && here is dangerous, as (annoyingly in this context), `false && true` is true (i.e. doesn't fail); use `;` and add `set -e`, or have two separate commands. We want to know when the umount fails, as it leaves the VM in an unclean state.

Reply:
Correct.